Drop-In/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if we need this file - it's English, but slightly different than the source English file. The translations repo had both the source file (which correlates to values/strings.xml) and this values-en file. Should we have both?

Reply:
It should be safe to delete. Android will default to `Drop-In/src/main/res/values/strings.xml` if a locale translation isn't specified.

Reply:
👍 Removed